index.html

```diff
@@ -1,56 +1,274 @@
+          <div>
+            <p class="client">CANOPY</p>
+          </div>
+          <div class="mark">
+            <img src="icons/Counter.png" alt="" />
+          </div>
+          <div>
+            <p class="role">Back End Dev</p>
+          </div>
+          <div class="mark">
+            <img src="icons/Counter.png" alt="" />
+          </div>
+          <div>
+            <p class="year">2015</p>
+          </div>
+        </div>
+        <div>
+          <p class="descrdetail">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+        </div>
+        <div class="tagcontainer">
+          <div class="tag">html</div>
+          <div class="tag">css</div>
+          <div class="tag">javaScript</div>
+        </div>
+        <div>
+          <button type="button" class="click">See Project</button>
+        </div>
+      </div>
+      <div class="c4">
+        <a href=""><img src="icons/portfolio4.png" alt="" class="portfoliocard" />
+        </a>
+        <div>
+          <p class="project4">Multi-Post Stories</p>
+        </div>
+        <div class="descrcontainer">
+          <div>
+            <p class="client">CANOPY</p>
+          </div>
+          <div class="mark">
+            <img src="icons/Counter.png" alt="" />
+          </div>
+          <div>
+            <p class="role">Back End Dev</p>
+          </div>
+          <div class="mark">
+            <img src="icons/Counter.png" alt="" />
+          </div>
+          <div>
+            <p class="year">2015</p>
+          </div>
+        </div>
+        <div>
+          <p class="descrdetail">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+        </div>
+        <div class="tagcontainer">
+          <div class="tag">html</div>
+          <div class="tag">css</div>
+          <div class="tag">javaScript</div>
+        </div>
+        <div>
+          <button type="button" class="click">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <div class="about-container">
+        <div>
+          <h2 class="about-title">About Myself</h2>
+        </div>
+        <div>
+          <p class="myself">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+        </div>
+        <div>
+          <p class="connect1">LET’S CONNECT</p>
+        </div>
+        <div class="networks">
+          <img src="icons/github.svg" alt="github" />
+          <img src="icons/Linkedin.svg" alt="linkedin" />
+          <img src="icons/manito.svg" alt="angellist" />
+          <img src="icons/twter.svg" alt="twiter" />
+          <img src="icons/m.svg" alt="medium" />
```

Comment:
- [ ] Kindly make sure that your icons are inside a `<ul>`. This will keep the code more organized and easier to understand for future readers.

Reply:
Thanks for the review, corrected.